trellis/core/subscriber.hpp

```diff
@@ -245,6 +244,7 @@ class SubscriberImpl {
   // Used to know how long to wait for the monitor layer
   time::TimePoint first_receive_time_{};
   bool did_receive_{false};
+  Timer watchdog_timer_{nullptr};
```

Comment:
Did you mean for this change to be in this PR?

Reply:
Yes, see my comment above.